thirtyinch/src/test/java/net/grandcentrix/thirtyinch/TiLifecycleObserverTest.java

```diff
@@ -249,6 +256,33 @@ public void onChange(final TiPresenter.State state,
     }
 
     @Test
+    public void testRemoveOtherObserver() throws Exception {
+        mPresenter.create();
+
+        final TiLifecycleObserver observer1 = mock(TiLifecycleObserver.class);
+        mPresenter.addLifecycleObserver(observer1);
+        final TiLifecycleObserver observer2 = mock(TiLifecycleObserver.class);
+        final Removable removable = mPresenter.addLifecycleObserver(observer2);
+
+        doAnswer(new Answer() {
+            @Override
+            public Object answer(final InvocationOnMock invocation) throws Throwable {
+                removable.remove();
+                return null;
+            }
+        }).when(observer1).onChange(any(TiPresenter.State.class), anyBoolean());
+
+        mPresenter.attachView(mock(TiView.class));
+
+        //receives both events
+        verify(observer1, times(2)).onChange(any(TiPresenter.State.class), anyBoolean());
```

Comment:
Are ok.. 👍 
But I would recommend to change the comment (https://github.com/grandcentrix/ThirtyInch/pull/94#discussion_r114708118)  with that info.. Instead of just "first event" and "second event"...

Maybe you can also improve the tests with the given State and Boolean value 🤔